cli-test/__snapshots__/cli.test.js.snap

```diff
@@ -49,10 +49,9 @@ Object {
     -r, --require <module>                      Module to preload
 
 Available scripts (camel or kebab case accepted)
-
-test - echo \"test script\"
-lint - echo \"lint.default\"
-lint.sub.thing - this is a description - echo \"deeply nested thing\"
+├─ test - echo \"test script\"
+└─ lint - echo \"lint.default\"
+   │  └─ lint.sub.thing - this is a description - echo \"deeply nested thing\"
```

Comment:
What's this extra line for? It looks kind of odd to me. Can we make that not appear like that?